sdk/core/core-client-rest/src/sendRequest.ts

```diff
@@ -163,9 +169,10 @@ function getRequestBody(body?: unknown, contentType: string = ""): RequestBody {
 
   switch (firstType) {
     case "multipart/form-data":
-      return isRLCFormDataInput(body)
-        ? { formData: processFormData(body) }
-        : { body: JSON.stringify(body) };
+      if (Array.isArray(body)) {
+        return { multipartBody: buildMultipartBody(body as PartDescriptor[]) };
```

Comment:
i may not know the whole context, it seems the contract is changed from { formData } to { multipartBody }, do we have other policies change?

Reply:
`formData` is still available, removing that API from core-rest-pipeline would be a breaking change for HLC. `multipartBody` is the lower level construct for generic multipart. It takes in an array of parts instead of `FormDataMap`.